frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -76,14 +78,28 @@ export const parseEncodedSnapshots = async (
                 }
             })
         } catch (e) {
-            posthog.capture('session recording had unparseable line', {
-                sessionId,
-                line: l,
-            })
-            captureException(e)
+            if (typeof l === 'string') {
```

Comment:
Could it be that the `l` isn't a `string` so it should be part of `unparseableLines`?

Reply:
It's possible... we call this from multiple places that each receive slightly different data so I'm trying to be as cautious as I can. TBH I just need to see examples periodically until my subconscious figures out why there are any unparseable lines 🤣